addon/components/base/bs-nav.js

```diff
@@ -65,6 +65,8 @@ import layout from 'ember-bootstrap/templates/components/bs-nav';
  You can override `tagName` if you want to use Bootstrap 4's ability to represent more structural
  components with `div` tags.
 
+ The `fill` styling is only available with Bootstrap 4
```

Comment:
When this is the case, it would be better to have the changes here be applied rather to the BS4-specific component (https://github.com/kaliber5/ember-bootstrap/blob/master/addon/components/bs4/bs-nav.js).

The setup might be a bit confusing at first, and I definitely should write a proper contributing guide. The idea is to have all common code in the `base` component, and alls version-specific modifications in the respective `bs3` or `bs4` components (that extend from base)

Reply:
Great thanks for the tips.  I think it should be set properly now.